docs/source/controllers/controller_configuration.rst

```diff
@@ -33,9 +33,9 @@ Temperature Sensors
 
 Fermentrack supports the DS18B20 temperature sensors, which use the OneWire protocol. This allows multiple sensors to be connected via a single pin on the controller board. For the ESP8622 board, this is the D6 pin. To ensure stable data transfers, you will also want to include a 4.7K resistor between the voltage source and the data pin. An example of how to gang multiple temperature sensors together, with resistor, looks like this:
 
-.. image:: https://www.microcontroller-project.com/uploads/2/2/1/5/22159166/nodemcu-esp8266-with-multiple-ds18b20-sensors_orig.png
+.. figure:: ds18b20-normal-power.jpg
 
-Again, for the ESP8622, you will want to connect the data wire to pin D6.
+    Image courtesy of https://www.tweaking4all.com/hardware/arduino/arduino-ds18b20-temperature-sensor/
```

Comment:
Might be good to change this to "Image courtesy of Tweaking4All" for succinctness, but we can check how it looks once the docs get rebuilt.

Reply:
Yeah, I had looked at that, I do think it is a little long. I originally had it just Tweaking4All, but he asked for a link to the article... I think we can just hyperlink it to the article, but use Tweaking4All as text.